pkg/rpcclient/waiter/waiter.go

```diff
@@ -15,11 +15,13 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/util"
 )
 
-// PollingBasedRetryCount is a threshold for a number of subsequent failed
-// attempts to get block count from the RPC server for PollingBased. If it fails
-// to retrieve block count PollingBasedRetryCount times in a raw then transaction
-// awaiting attempt considered to be failed and an error is returned.
-const PollingBasedRetryCount = 3
+const (
```

Comment:
Not really needed, we only have one constant.

Reply:
Fixed, also left from the old version of this PR.